api/openapi-spec/swagger.json

```diff
@@ -13110,22 +13110,22 @@
+          "description": "List of ingress rules to be applied to the selected pods if `policyTypes` includes \"Ingress\". If this field is empty then this NetworkPolicy does not allow any incoming flows (and serves solely to ensure that the pods it selects deny ingress by default).",
           "items": {
             "$ref": "#/definitions/io.k8s.api.networking.v1.NetworkPolicyIngressRule"
           },
           "type": "array"
         },
         "podSelector": {
           "$ref": "#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.LabelSelector",
-          "description": "Selects the pods to which this NetworkPolicy object applies. The array of ingress rules is applied to any pods selected by this field. Multiple network policies can select the same set of pods. In this case, the ingress rules for each are combined additively. This field is NOT optional and follows standard label selector semantics. An empty podSelector matches all pods in this namespace."
+          "description": "Selects the pods to which this NetworkPolicy object applies. A network flow is bidirectional but initiated in one direction, from a source to a destination. The whole flow is allowed or not depending on whether the initiation is allowed. A network flow with a pod on either or both ends is allowed to be initiated if both (a) initiation from the source is allowed and (b) initiation to the destination is allowed.\n\nInitiation from a source is allowed if one of the following is true. - The source is not a pod. - The source is a pod and there is at least one NetworkPolicy whose `podSelector`\n  selects the pod, `policyTypes` includes \"Egress\",\n  and `egress` includes a rule that allows the initiation.\n- The source is a pod and there is no NetworkPolicy whose `podSelector` selects\n  the pod and whose `policyTypes` includes \"Egress\".\n\nInitiation to a destination is allowed if at least one of the following is true. - The destination is not a pod. - The destination is a pod and the source is the pod's node. - The destination is a pod and there is at least one NetworkPolicy whose\n  `podSelector` selects the pod, `policyTypes` includes \"Ingress\",\n  and `ingress` includes a rule that allows the initiation.\n- The destination is a pod and there is no NetworkPolicy whose `podSelector`\n  selects the pod and whose `policyTypes` includes \"Ingress\".\n\nThis field is NOT optional and follows standard label selector semantics. An empty podSelector matches all pods in this namespace."
```

Comment:
The rendering into the string here is pretty sucky.  is there any way to write structure in the source comments that will be nicely rendered here?